src/organisms/organism.py

```diff
@@ -29,37 +29,37 @@ def __init__(self, config, x_pos, y_pos, nn = False):
                         (0, 1),
                         (1, 1)] # used as inputs for the move function, one element for moving to one of the adjacent tiles
 
-    def eat(self, config, world):
-        world[self.position.y][self.position.x] = Empty(config, self.position.x, self.position.y)
-        self.lifetime += config.food_bonus
+    def eat(self, inputs, world):
+        world[self.position.y][self.position.x] = Empty(inputs["size"]["tile_width"], self.position.x, self.position.y)
+        self.lifetime += inputs["food"]["lifetime_bonus"]
 
-    def move(self, config, direction, world):
-        if 0 <= self.position.x + direction[0] < config.row_length and 0 <= self.position.y + direction[1] < config.column_length:
+    def move(self, inputs, direction, world):
+        if 0 <= self.position.x + direction[0] < inputs["world"]["size"]["row_length"] and 0 <= self.position.y + direction[1] < inputs["world"]["size"]["column_length"]:
             if not world[self.position.y + direction[1]][self.position.x + direction[0]].has_organism:
                 self.position.x += direction[0]
                 self.position.y += direction[1]
                 if not direction == (0, 0):
-                    self.score += config.move_bonus
+                    self.score += inputs["population"]["interactions"]["move_bonus"]
         else:
-            self.lifetime -= config.border_damage
+            self.lifetime -= inputs["population"]["interactions"]["border_damage"]
```

Comment:
Just to throw it in there: do we still want to take damage at the border?
I think we once talked about "wrapping around", so the organisms can move "freely"...
-> What did we decide there?

Reply:
I would leave it for now, maybe change it after having the branch and the interface together...